node/src/chain_spec.rs

```diff
@@ -507,6 +518,11 @@ fn amplitude_genesis(
 			),
 	));
 
+	let token_balances = balances
+		.iter()
+		.flat_map(|k| vec![(k.0.clone(), XCM(DOT), 1 << 60), (k.0.clone(), XCM(KSM), 1 << 60)])
```

Comment:
We probably don't want to hand out these tokens for free. So let's set the token balances to something empty for all assets.

Reply:
done